crates/core/src/keepers/common.rs

```diff
@@ -80,4 +86,46 @@ impl Keeper {
         dbg!(result);
         Ok(())
     }
+
+    /// Execute a withdrawal.
+    /// # Arguments
+    /// * `deposit_key` - The deposit key.
+    /// * `set_prices_params`.
+    pub async fn execute_withdrawal(
+        &self,
+        deposit_key: &str,
```

Comment:
should rename `deposit_key` by `withdrawal_key` [cairo withdrawal handler](https://github.com/keep-starknet-strange/satoru/blob/main/src/exchange/withdrawal_handler.cairo#L39)

Reply:
Updated!